airbyte-integrations/connectors/source-azure-table/source_azure_table/spec.json

```diff
@@ -17,14 +17,14 @@
       "storage_access_key": {
         "title": "Access Key",
         "type": "string",
-        "description": "The access key to your storage account. Read more about access keys here - https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal#view-account-access-keys",
+        "description": "Azure Table Storage Access Key. See the <a href=\"https://docs.airbyte.io/integrations/sources/azure-table\">docs</a> for more information on how to obtain this key.",
         "order": 1,
         "airbyte_secret": true
       },
       "storage_endpoint_suffix": {
         "title": "Endpoint Suffix",
         "type": "string",
-        "description": "The Table service account URL suffix. Read more about suffixes here - https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#create-a-connection-string-with-an-endpoint-suffix",
```

Comment:
Please, update `airbyte/docs/integrations/sources/azure-table.md` and set this information to `md` file